src/wallet/puzzles/recompile-all.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# This hack is a quick way to recompile everything in this directory
+
+#BASE_DIR=`pwd | dirname`
```

Comment:
per my message there are a couple of errors here that ShellCheck is going to raise in lint. https://www.shellcheck.net/

Reply:
Thanks for that.